src/tools/README.txt

```diff
@@ -40,4 +40,4 @@ upgrader:	Automated upgrade mechanism.
 
 virt:		Source to our "permabit-virt" Debian package: for 
 		manipulating a virtual network interface, used in
-		some automated tests.
+		some automated tests. More and More.
```

Comment:
Confusing change.  One could infer that it's meant to represent more automated tests; if so, it would be better to be specific.